doc/source/getting_started/qtcrashcourse.rst

```diff
@@ -91,7 +91,93 @@ http://qt-project.org/doc/qt-4.8/graphicsview.html#the-graphics-view-coordinate-
 Mouse and Keyboard Input
 ------------------------
 
+Events Overview
+^^^^^^^^^^^^^^^
+Understanding events in a Qt application is fundamental before delving into specific input handling such as mouse and keyboard:
 
+- Events in Qt are conceptualized as user interactions with the application, each represented by an event object (QEvent).
+- Various types of events correspond to different user interactions.
+- Event objects encapsulate details concerning the specific occurrence.
+- Dispatched to designated event handlers within the widget where the interaction occurs, these events allow for customizable responses.
+- Handlers may be extended or redefined to modify widget response to interactions.
+
+Mouse Events
+^^^^^^^^^^^^
+Interactions such as clicks, movements, and button releases are managed by overriding methods including ``mousePressEvent``, ``mouseReleaseEvent``, ``mouseDoubleClickEvent``, and ``mouseMoveEvent``.
+
+Integration with PyQtGraph
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+PyQtGraph utilizes QWidget subclasses to present graphics and plots. Consequently, the event-handling methods discussed can be directly integrated into PyQtGraph widgets. This integration enables sophisticated interactive features in applications that leverage PyQtGraph for visual data representation.
+
+Example: Handling Mouse Clicks in a PlotWidget::
+
+    from PyQt6.QtWidgets import QApplication, QMainWindow # Should work with PyQt5 / PySide2 / PySide6 as well
+    from PyQt6.QtCore import Qt
+
+    class MainWindow(QMainWindow):
+        def __init__(self):
+            super().__init__()
+            self.setWindowTitle('Mouse and Keyboard Event Demo') # Sets the Title of the window
+            self.setGeometry(100, 100, 400, 300) # Sets the position and size of the window
+
+        def mousePressEvent(self, event): # This method checks if the left mouse button was pressed on the widget and prints the position of the click.
```

Comment:
There is a horizontal scrolling bar, try and keep the line-width here really short. In the rendered docs it looks like:

<img width="757" alt="image" src="https://github.com/pyqtgraph/pyqtgraph/assets/646398/a95ca014-e58f-42ed-9274-c2f16828c45e">

Reply:
Hi [j9ac9k](https://github.com/j9ac9k),

Thank you for your feedback!

I wanted to clarify your recommendation about the line length. Does the 88-character limit apply only to the code lines, or should I also adjust the text in the documentation to adhere to this limit?

Thanks again for your guidance!
Lamis

Reply:
I try to conform the 88 character line length to everything is rst files.
Sphinx will ignore single newline characters so adding new lines does not
impact the rendered result, it just makes it easier to work with in a text
editor.

I’m sure there are places where this line limit won’t make sense (like
linking to something that had a long link) so there is room for exceptions.